### PR TITLE
move toolbars content to new components

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

File: ui/client/components/toolbars/AttachmentsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {CollapsibleToolbar} from "../toolbarComponents/CollapsibleToolbar"
+import ProcessAttachments from "../ProcessAttachments"
+import React from "react"
+import {useTranslation} from "react-i18next"
+
+export function AttachmentsPanel() {
+  const {t} = useTranslation()
+
+  return (
+    <CollapsibleToolbar id="ATTACHMENTS-PANEL" title={t("panels.attachments.title", "Attachments")}>
```
Comment (PPiedel) [inline ui/client/components/toolbars/AttachmentsPanel.tsx]: why capital letters?

File: ui/client/components/toolbars/edit/EditPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+function EditPanel(props: Props) {
+  const capabilities = useSelector(getCapabilities)
+  const {selectionActions} = props
+  const {t} = useTranslation()
+
+  return (
+    <CollapsibleToolbar id="EDIT-PANEL" title={t("panels.edit.title", "Edit")}>
+      <ToolbarButtons small>
+        {capabilities.write ? <Undo/> : null}
```
Comment (PPiedel) [inline ui/client/components/toolbars/edit/EditPanel.tsx]: One `capabilities.write ` wouldn't be enough?

File: ui/client/components/tips/Tips.js [hunk 3]
```diff
@@ -29,20 +32,36 @@ export class Tips extends Component {
...
-                      hasNeitherErrorsNorWarnings={ProcessUtils.hasNeitherErrorsNorWarnings(currentProcess)}/>}
-          {!ProcessUtils.hasNoErrors(currentProcess) && <Errors errors={errors}
-                                                                showDetails={this.showDetails}
-                                                                currentProcess={currentProcess}/>}
-          {!ProcessUtils.hasNoWarnings(currentProcess) && <Warnings warnings={ProcessUtils.extractInvalidNodes(warnings.invalidNodes)}
-                                                                    showDetails={this.showDetails}
-                                                                    currentProcess={currentProcess}/>}
-        </Scrollbars>
-      </div>
+      <CollapsibleToolbar title={i18next.t("panels.tips.title", "Tips")} id="TIPS-INFO">
```
[the remark this answers is not in the document]
Comment (JulianWielga) [reply ui/client/components/tips/Tips.js]: ?

File: ui/client/components/toolbars/AttachmentsPanel.tsx (hunk 1 above)
Comment (JulianWielga) [reply ui/client/components/toolbars/AttachmentsPanel.tsx]: to differentiate from string that should be translated and avoid eslint-disable

File: ui/client/components/toolbars/edit/EditPanel.tsx (hunk 2 above)
Comment (JulianWielga) [reply ui/client/components/toolbars/edit/EditPanel.tsx]: yes

**Comment by arkadius** [comment]
Can you change a bit order of buttons to be more intuitive from user perspective?
- status: save, deploy, cancel, metrics
- view: business, zoom-in, zoom-out, reset
- edit: undo, redo, cut, copy, paste, layout

**Comment by arkadius** [comment]
Can you asssign all left panels to top-left corner in initial configuration? It is more likely the same behaviour like it was before change - let's not change it to not confuse users

**Comment by arkadius** [comment]
Tips are flickering after hover on inactive button:
Screencast from 16 03 2020 17:47:21 webm

**Comment by arkadius** [comment]
Test result5: After grouping nodes, error state of property button is lost:
Screencast from 16 03 2020 18:16:59 webm

**Comment by JulianWielga** [comment]
> Test result4: Tips are flickering after hover on inactive button:
> Screencast from 16 03 2020 17:47:21 webm

I didn't change this behavior. it has already acted this way

**Pull request merged**